src/XrdClCurl/configs/xrdcl-curl-plugin.conf

```diff
@@ -0,0 +1,5 @@
+
+url = http://*;https://*
+lib = ${LIB_INSTALL_DIR}/libXrdClCurl${LIBRARY_SUFFIX}
```

Comment:
This can be simply `libXrdClCurl.so` to avoid needing to call `configure_file()`.
XRootD is configured with an `RPATH` that preferentially picks up plugins from the same directory as the rest.